addon/templates/components/bs3/bs-accordion/body.hbs

```diff
@@ -1,5 +1,5 @@
-{{#component collapseComponent ariaRole="tabpanel" collapsed=collapsed class="panel-collapse"}}
-  <div class="panel-body {{class}}">
+{{#component this.collapseComponent ariaRole="tabpanel" collapsed=@collapsed class="panel-collapse"}}
```

Comment:
Shouldn't this be `aria-role="tabpanel"` as the HTML attribute is name `aria-role` and not `ariaRole`?

Reply:
As the component helper does not allow to be used in a angle brackets like fashion (it's not a component), we cannot set HTML attributes in the way we could w/ angle bracket components. So this sets the `ariaRole` *property* of `Ember.Component` (see https://api.emberjs.com/ember/3.10/classes/Component/properties/ariaRole?anchor=ariaRole), that is mapped to the `role` attribute (through our beloved `attributeBindings`) internally.

Reply:
Thanks a lot for this detailed explanation. :+1: